src/twisted/internet/tcp.py

```diff
@@ -934,8 +935,13 @@ def __exit__(excType, excValue, traceback):
         """
 
 
+class _HasClose(typing_extensions.Protocol):
+    def close(self) -> object:
```

Comment:
Not `-> None`?

Reply:
It's less restrictive, the consumer doesn't care about the return value at all